jest.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  transform: {
+    '^.+\\.(t|j)sx?$': '@swc/jest',
+  },
+  testEnvironment: 'jsdom',
+  testRegex: 'tests/.*test\\.tsx?$',
+  collectCoverage: true,
+  collectCoverageFrom: ['src/**/*.ts'],
```

Comment:
should we add reporters 
```suggestion
  collectCoverageFrom: ['src/**/*.ts'],
  coverageReporters: ['html'],
```